conductr_cli/bndl_docker.py

```diff
@@ -182,14 +182,21 @@ def docker_unpack(destination, data, is_dir, maybe_name, maybe_tag):
         os.makedirs(os.path.join(destination, 'blobs/sha256'))
         os.makedirs(os.path.join(destination, 'refs'))
 
-        file_write_bytes(os.path.join(destination, 'oci-layout'), '{"imageLayoutVersion": "1.0.0"}'.encode('UTF-8'))
-
         layers_to_digests = {}
         symlinks = {}
         digests = {}
         sizes = {}
+        first_mtime = [0]
+        is_first_entry = [True]
+
```

Comment:
This weirdness `[0]` and `[True]` is to get around python's weird scoping rules.